app/src/main/java/pl/jakubokrasa/kursakademiaandroida/core/data/api/RickAndMortyDatabase.kt

```diff
@@ -0,0 +1,17 @@
+package pl.jakubokrasa.kursakademiaandroida.core.data.api
+
+import androidx.room.Database
+import androidx.room.RoomDatabase
+import androidx.room.TypeConverters
+import pl.jakubokrasa.kursakademiaandroida.core.data.local.database.ListConverter
+import pl.jakubokrasa.kursakademiaandroida.core.data.local.model.OriginCached
+import pl.jakubokrasa.kursakademiaandroida.features.character.data.CharacterCached
+import pl.jakubokrasa.kursakademiaandroida.features.episode.data.local.EpisodeDao
+import pl.jakubokrasa.kursakademiaandroida.features.episode.data.local.model.EpisodeCached
+import pl.jakubokrasa.kursakademiaandroida.features.location.data.local.model.LocationCached
+
+@Database(entities = [EpisodeCached::class, CharacterCached::class, LocationCached::class, OriginCached::class], version = 1)
```

Comment:
O ile dobrze kojarzę, jeśli jeden obiekt zawiera się w drugim, a nie jest bezpośrednio wrzucany do rooma, to nie ma potrzeby definiować go tutaj jako entity.